hera_pspec/pspecdata.py

```diff
@@ -926,7 +1215,27 @@ def rephase_to_dset(self, dset_index=0, inplace=True):
 
         if inplace is False:
             return dsets
-            
+
+
+def get_delays(freqs):
```

Comment:
Should this go in the `utils` file perhaps?

Reply:
yeah, the `get_delays` should probably go in `utils`, b/c it has as possibly broader use-case beyond just `pspecdata`..